examples/basics/thermal_basics.py

```diff
@@ -9,123 +9,173 @@
+#
+# .. note::
+#    This will also disable crucial popups, for example prompts to save data or overwrite data, or
+#    dialogs used to reconcile differences in material data between the database and MOT file. In
+#    each case the default action will be taken. This setting will persist until Motor-CAD is
+#    closed.
+mc.set_variable("MessageDisplayState", 2)
 
 # %%
-# Open relevant file
+# Open template file
 # ~~~~~~~~~~~~~~~~~~
-# Specify the working directory and open the relevant file for the
-# thermal analysis.
-working_folder = os.getcwd()
-mcad.load_template("e8")
-mcad_name = "e8_mobility"
-mcad.save_to_file(os.path.join(working_folder, mcad_name))
-
-mcad.load_from_file(os.path.join(working_folder, mcad_name + ".mot"))
-
-print("Initialization completed.")
+# This example is based on the e8 IPM motor template. Open the template and save the file to the
+# temporary folder.
+mc.load_template("e8")
+mc.save_to_file(working_folder + "/" + mot_name + ".mot")
 
 # %%
-# Create analysis
-# ---------------
-# Creating the analysis consists of showing the thermal context, displaying
-# the **Scripting** tab, setting parameters, and saving the file.
-
-# Show thermal context
-# -----------------------
-mcad.show_thermal_context()
-
-# %%
-# Display the **Scripting** tab.
-mcad.display_screen("Scripting")
-
-# %%
-# Change the housing diameter.
-mcad.set_variable("Housing_Dia", 250)
-
-# %%
-# Set the flow rate of the WJ fluid volume.
-mcad.set_variable("WJ_Fluid_Volume_Flow_Rate", 0.002)
-
-# %%
-# Set the temperature of the WJ fluid inlet.
-mcad.set_variable("WJ_Fluid_Inlet_Temperature", 25)
-
-# %%
-# Change the cooling fluid.
-mcad.set_fluid("HousingWJFluid", "Dynalene HF-LO")
+# Set Motor-CAD to Thermal
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Show the Thermal context in Motor-CAD and ensure that the BPM
+# Motor Type is set.
+mc.show_thermal_context()
+mc.set_variable("Motor_Type", 0)
+print("Initialisation completed.")
 
 # %%
-# Set the heat transfer correlation.
-mcad.set_variable("Calc/Input_h[WJ]_Rear_Housing", 1)
-mcad.set_array_variable("HousingWJ_CalcInputH_A", 0, 1)
-
-wj_fluid_k = mcad.get_variable("WJ_Fluid_Thermal_Conductivity")
-wj_fluid_rho = mcad.get_variable("WJ_Fluid_Density")
-wj_fluid_mu = mcad.get_variable("WJ_Fluid_Dynamic_Viscosity")
-wj_fluid_u_a = mcad.get_array_variable("HousingWJ_Velocity_A", 0)
-wj_fluid_u_r = mcad.get_variable("WJ_Channel_Fluid_Velocity_[Rear]")
+# Set up analysis
+# ---------------
+# Setting up the analysis consists of setting the geometry and thermal input data parameters and
+# saving the file.
+#
+# Geometry setup
+# ~~~~~~~~~~~~~~
+# Display the **Scripting** tab and set the housing outer diameter to 250 mm.
+#
+# .. note::
+#    To set parameter values via automation with the GUI visible, a different tab must be displayed.
+#    The **Scripting** tab is chosen because no parameters on this tab will be changed in this
```

Comment:
```suggestion
# .. note::
#    To set parameter values by automation with the GUI visible, a different tab must be shown.
#    The **Scripting** tab is chosen because no parameters on this tab are changed in this
```